cypress/integration/trends.js

```diff
@@ -6,7 +6,7 @@ describe('Trends', () => {
         cy.location('pathname').should('include', '/edit')
     })
 
-    it('Can load a graph from a URL directly', () => {
+    it.skip('Can load a graph from a URL directly', () => {
```

Comment:
This is A LOT of Volkswagen, are you sure about this?

Reply:
Trying The Toyota Way this time...